packages/atlas/src/components/_overlays/MemberDropdown/MemberDropdownList.tsx

```diff
@@ -44,16 +47,22 @@ export const MemberDropdownList: FC<MemberDropdownListProps> = ({
         />
       </SwitchMemberItemListContainer>
       <SectionContainer>
-        {sortedMemberChannels.map((channel) => (
-          <ListItem
-            key={channel.id}
-            onClick={() => onChannelChange?.(channel.id)}
-            nodeStart={<Avatar assetUrls={channel.avatarPhoto?.resolvedUrls} size={32} />}
-            label={channel?.title ?? ''}
-            caption={channel ? `${channel?.followsNum} followers` : undefined}
-            selected={type === 'channel' ? channel.id === channelId : false}
-          />
-        ))}
+        {sortedMemberChannels.map((channel) => {
+          const isSelected = type === 'channel' ? channel.id === channelId : false
```

Comment:
`const isSelected = type === 'channel' && channel.id === channelId` will work the same way here?